packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.hooks.tsx

```diff
@@ -184,64 +186,66 @@ export const useYppGoogleAuth = ({
-            errorResponseData.errorCode === YppAuthorizationErrorCode.YOUTUBE_QUOTA_LIMIT_EXCEEDED
-
-          if (isQuotaError) {
-            displaySnackbar({
-              title: 'Something went wrong',
-              description:
-                "Due to high demand, we've reached the quota on the daily new sign ups. Please try again tomorrow.",
-              iconType: 'error',
-            })
-            closeModal()
-            return
-          }
-
-          const isChannelNotFoundError =
-            errorResponseData &&
-            'errorCode' in errorResponseData &&
-            errorResponseData.errorCode === YppAuthorizationErrorCode.CHANNEL_NOT_FOUND
-
-          if (isChannelNotFoundError) {
-            displaySnackbar({
-              title: 'Authorization failed',
-              description: `You don't have a YouTube channel.`,
-              iconType: 'error',
-            })
-            setYtRequirmentsErrors(Object.values(YppAuthorizationErrorCode))
-            onChangeStep('requirements')
-            return
-          }
-
-          const isChannelAlreadyRegistered =
-            errorResponseData &&
-            'errorCode' in errorResponseData &&
-            errorResponseData.errorCode === YppAuthorizationErrorCode.CHANNEL_ALREADY_REGISTERED
-
-          if (isChannelAlreadyRegistered) {
-            const { data } = await client.query<GetFullChannelQuery, GetFullChannelQueryVariables>({
-              query: GetFullChannelDocument,
-              variables: { where: { id: errorResponseData.result.toString() } },
-            })
-            setAlreadyRegisteredChannel({
-              channelTitle: data.channelByUniqueInput?.title || '',
-              ownerMemberHandle: data.channelByUniqueInput?.ownerMember?.handle || '',
-            })
-
-            onChangeStep('channel-already-registered')
-            return
+          if (errorResponseData && 'code' in errorResponseData) {
+            switch (errorResponseData.code) {
+              case YppAuthorizationErrorCode.YOUTUBE_QUOTA_LIMIT_EXCEEDED:
+                displaySnackbar({
+                  title: 'Something went wrong',
+                  description:
+                    "Due to high demand, we've reached the quota on the daily new sign ups. Please try again tomorrow.",
+                  iconType: 'error',
+                })
+                closeModal()
+                return
+              case YppAuthorizationErrorCode.CHANNEL_NOT_FOUND:
+                displaySnackbar({
+                  title: 'Authorization failed',
+                  description: `You don't have a YouTube channel.`,
+                  iconType: 'error',
+                })
+                setYtRequirementsErrors(Object.values(YppAuthorizationErrorCode))
+                onChangeStep('requirements')
+                return
+              case YppAuthorizationErrorCode.CHANNEL_ALREADY_REGISTERED:
+                // eslint-disable-next-line no-case-declarations
```

Comment:
Just use curly braces and this should fix eslint problem. I would avoid `eslint-disable-next-line` as much as I can